public/index.php

```diff
@@ -11,28 +11,37 @@
     </section>
 
     <section class="col-lg-8 col-11 holding-form">
-      <form class="form-inline index-form">
-        <label class="sr-only" for="location">Location</label>
-        <div class="input-group col-lg-4 col-sm-12">
+      <form class="form-inline index-form" method="post" action="search.php">
+        <?php require_once('./includes/find-house.php') ?>
```

Comment:
I see no reason why we have to include our response file here since the database call will always be made when the page is refreshed (Not the intended behavior here, since we don't want to overload the database when we have lots of people visiting the index page). Can we build the form with Javascript and only make database calls when it is necessary.